core/src/main/resources/xml/xsl/adapter2mermaid.xsl

```diff
@@ -0,0 +1,535 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xsl:stylesheet xmlns:xsl="http://www.w3.org/1999/XSL/Transform" xmlns:xs="http://www.w3.org/2001/XMLSchema" version="2.0">
+	<xsl:output method="text" indent="yes" omit-xml-declaration="yes"/>
+
+	<xsl:variable name="errorForwards" select="('exception','failure','fail','timeout','illegalResult','presumedTimeout','interrupt','parserError','outputParserError','outputFailure')"/>
+	<xsl:variable name="adapterCount" select="count(//adapter)"/>
+
+	<xsl:template match="/">
+		<!-- Create the Mermaid graph in 2 steps
+			- First preprocess adapters, putting pipes in the correct order, explicitly adding implicit forwards and preprocess input and output validators and wrappers
+			- Then convert the adapter to mermaid code-->
+		<xsl:variable name="preproccessedConfiguration">
+			<xsl:apply-templates select="*" mode="preprocess"/>
+		</xsl:variable>
+
+		<xsl:text>
+			&lt;div class="inmodal">
+				&lt;div class="modal-body" ng-if="!error">
+					&lt;ng-mermaid>
+		</xsl:text>
```

Comment:
dit kan echt niet, het framework moet een goede, mooie mermaid flow aanbieden, zonder frontend dingen er in.